src/overlays/actors/ovl_En_Encount1/z_en_encount1.c

```diff
@@ -28,10 +27,129 @@ const ActorInit En_Encount1_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+static s16 sActorList[] = {
+    ACTOR_EN_GRASSHOPPER,
+    ACTOR_EN_WALLMAS,
+    ACTOR_EN_PR2,
+    ACTOR_EN_PR2,
+};
+
+static s16 sActorParams[] = { 1, 0, 1, 3 };
+
+void EnEncount1_Init(Actor* thisx, PlayState* play) {
+    EnEncount1* this = THIS;
+
+    if (this->actor.params <= 0) {
+        Actor_MarkForDeath(&this->actor);
+        return;
+    }
+
+    this->actorType = ENENCOUNT1_GET_TYPE(&this->actor);
+    this->unk_14C = ENENCOUNT1_GET_7C0(&this->actor);
+    this->unk_154 = ENENCOUNT1_GET_PATH(&this->actor);
+    this->unk_158 = this->actor.world.rot.x;
+    this->unk_15C = this->actor.world.rot.y;
+    this->unk_160 = (this->actor.world.rot.z * 40.0f) + 120.0f;
+
+    if (this->unk_154 >= 0x3F) {
+        this->unk_154 = -1;
+    }
+    if (this->actor.world.rot.z < 0) {
+        this->unk_160 = -1.0f;
+    }
+    if (this->actorType == EN_ENCOUNT1_SKULLFISH_2) {
+        this->unk_15A = ENENCOUNT1_GET_PATH(&this->actor);
+        this->path = SubS_GetPathByIndex(play, this->unk_15A, 0x3F);
+        this->unk_154 = -1;
+        this->unk_160 = -1.0f;
+    }
+    this->actor.flags &= ~ACTOR_FLAG_1;
+    this->actionFunc = func_808E0954;
+}
+
+void func_808E0954(EnEncount1* this, PlayState* play) {
+    Player* player = GET_PLAYER(play);
+    Vec3f spawnPos;
+    f32 sp64;
+    f32 temp_fv0_2;
+    s16 sp5E;
+    s16 actorList;
+    s32 actorParams;
+    CollisionPoly* sp54;
+    s32 sp50;
+
+    if (((this->unk_14E >= this->unk_14C) || ((0.0f < this->unk_160) && (this->unk_160 < this->actor.xzDistToPlayer)) ||
+         ((0 < this->unk_154) && (this->unk_154 <= this->unk_152)))) {
```

Comment:
```suggestion
    if (((this->unk_14E >= this->unk_14C) || ((this->unk_160 > 0.0f) && (this->unk_160 < this->actor.xzDistToPlayer)) ||
         ((this->unk_154 > 0) && (this->unk_154 <= this->unk_152)))) {
```
Slight nit, if this still matches I think looks a little cleaner